internal/prometheus/prometheus.go

```diff
@@ -196,3 +200,31 @@ func (c *captureStatusResponseWriter) WriteHeader(statusCode int) {
 	c.ResponseWriter.WriteHeader(statusCode)
 	c.status = statusCode
 }
+
+func prettyByteSize(b uint64) string {
+	bf := float64(b)
+	for _, unit := range []string{"", "K", "M", "G", "T", "P", "E", "Z"} {
+		if math.Abs(bf) < 1000.0 {
+			return fmt.Sprintf("%3.1f%sB", bf, unit)
+		}
+		bf /= 1000.0
+	}
+	return fmt.Sprintf("%.1fYiB", bf)
+}
+
+func allocHandler(rsp http.ResponseWriter, req *http.Request) {
+	var m runtime.MemStats
+	runtime.ReadMemStats(&m)
+
+	total := m.HeapInuse + m.StackInuse + m.MCacheInuse + m.MSpanInuse
```

Comment:
Can you help me understand why we these metrics and not something like `HeapAlloc`?

Reply:
`Alloc`/`HeapAlloc` will show a lower number, moving the metrics reported here further away from what other tools report.